lib/ansible/module_utils/net_tools/nios/api.py

```diff
@@ -55,25 +54,20 @@
 NIOS_SRV_RECORD = 'record:srv'
 NIOS_NAPTR_RECORD = 'record:naptr'
 NIOS_TXT_RECORD = 'record:txt'
-NIOS_NSGROUP = 'nsgroup'
```

Comment:
Seems like your branch is not updated, else why have you removed the support for the existing modules.